JS/models/classificationLevel.js

```diff
@@ -1,8 +1,8 @@
-const ClassificationLevel  = {
-    UNCLASSIFIED: 'Unclassified',
-    CONFIDENTIAL: 'Confidential',
-    SECRET: 'Secret',
-    TOP_SECRET: 'Top_secret'
+const ClassificationLevel = {
+  Unclassified: "Unclassified",
```

Comment:
Please use CAPITAL_LETTER notation for both constants and constant properties (if it is an object like it is in your case).